pykafka/simpleconsumer.py

```diff
@@ -64,6 +64,7 @@ def __init__(self,
                  auto_commit_interval_ms=60 * 1000,
                  queued_max_messages=2000,
                  fetch_min_bytes=1,
+                 fetch_error_backoff_ms=500,
```

Comment:
Yes, come to think of it it's probably a better idea to keep these interfaces fully synchronized. The problem is that you end up with a bunch of parameters that don't do anything for one or the other implementation, which is a pretty major API readability issue.